ui/app/src/CHANGELOG.md

```diff
@@ -36,10 +36,15 @@
   * `actions/dialogs`: Renamed `updateEditConfig` action to `updateEditDialogConfig`
 * [hooks]
   * `usePreviewUrlControl`: Removed `history` prop. Retrieval of search and navigate (previously called 'push') is now done internally.
+  * Removed `useLogicResource` hook.
+  * Removed `useSelectorResource` hook.
+  * Removed `useQuickCreateListResource` hook.
 * Removed deprecated `aws-file-upload` and `aws-video` upload controls.
 * Migrated the Studio UI build to Vite/SWC
 * Rollup's XB build to use SWC
 * Upgraded target compilation to ES2022, dropping many code transforms for features that are supported by most modern browsers such as nullish coalescing, optional chaining, object spreading and destructuring.
+* Removed `PreviewPageExplorerPanel` component.
```

Comment:
This doesn't seem to have been removed? Nor should it.

Reply:
this was incorrect, removing